worker-languagedetection-container/testcases/automated/CAF_4001/cherokee.yaml.testcase

```diff
@@ -54,43 +54,43 @@ expectedOutputData:
       DetectedLanguage1_Code:
         action: "replace"
         values:
-        - data: "chr"
+        - data: "unknown"
       DetectedLanguage1_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "99"
+        - data: "63"
```

Comment:
Significant change here, cannot identify cherokee anymore. If it is not supported then remove this testcase and make sure that cherokee is not referred to as a supported language.